datahub-web-react/src/app/search/SearchPage.tsx

```diff
@@ -1,131 +1,12 @@
 import React from 'react';
```

Comment:
wondering if we should rename this component `Homepage`?

Reply:
I'm fine with either

Reply:
renaming